portfolio/src/main/java/com/google/sps/servlets/FormServlet.java

```diff
@@ -0,0 +1,75 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns nametag text. */
+@WebServlet("/handle-comment")
+public class FormServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Retrieve newest user comment and store it as a Comment Entity
+    String userComment = request.getParameter("user-comment");
+    Entity commentEntity = new Entity("Comment");
+    commentEntity.setProperty("text", userComment);
+
+    // Store the Comment Entity in Datastore 
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
How does the servlet know which database and table to connect to? Is it stored in some config file?

Reply:
I accidentally forgot to include a commit of my updated pom.xml file--I included a dependency to Google AppEngine here. Regarding specific tables, to my understanding Datastore just keeps track of user-created groups of Entities. Please let me know if you want me to include more specific implementation information within my comments or take any other steps!

Reply:
That sounds good, was just curious